packages/vue-instantsearch/src/__tests__/common.test.js

```diff
@@ -78,8 +78,8 @@ createHierarchicalMenuTests(async ({ instantSearchOptions, widgetParams }) => {
 });
 
 createBreadcrumbTests(async ({ instantSearchOptions, widgetParams }) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { transformItems, ...hierarchicalWidgetParams } = widgetParams;
```

Comment:
maybe a clarifying comment can be useful to why we don't support transformItems here?

Reply:
Addressed in f843104ff.